### PR TITLE
Fix missing float cmp for XIP

### PR DESCRIPTION
**Pull request by no1wudi**

### CODE REVIEW

File: core/iwasm/compilation/aot_emit_numberic.c [hunk 1]
```diff
@@ -234,15 +234,49 @@ compile_op_float_min_max(AOTCompContext *comp_ctx, AOTFuncContext *func_ctx,
         nan = LLVMConstRealOfString(ret_type, "NaN");
     char *intrinsic = is_min ? (is_f32 ? "llvm.minnum.f32" : "llvm.minnum.f64")
                              : (is_f32 ? "llvm.maxnum.f32" : "llvm.maxnum.f64");
+    bool is_i32 = is_f32;
```
Comment (wenyongh) [inline core/iwasm/compilation/aot_emit_numberic.c:237]: Seems that `is_i32` isn't used?

**Review by wenyongh — COMMENTED** [review]
LGTM

File: core/iwasm/compilation/aot_emit_numberic.c (hunk 1 above)
Comment (no1wudi) [reply core/iwasm/compilation/aot_emit_numberic.c:237]: Emm, is_i32 used in macro `LLVM_BUILD_OP_OR_INTRINSIC`

Comment (wenyongh) [reply core/iwasm/compilation/aot_emit_numberic.c:237]: Got it, thanks.

**Pull request merged**